+++
date = 2018-08-06T00:00:00   # Schedule page publish date.
title = "Workshop: Population Genomics in R"
authors = ["Nik Gr√ºnwald", "Brian Knaus", "Zachary Foster"]
time_start = 2018-07-28T13:00:00
time_end = 2018-07-28T17:00:00
abstract = "<img src = '/img/learning/popgen2.png' align=right>Analysis of population genetic data remains challenging, particularly in the genomics era. High throughput sequencing has resulted in the opportunity to sequence variants (SNPs, indels, etc.) at the genome scale. However, application of these technologies for the characterization of pathogens provides unique challenges. This [workshop](http://grunwaldlab.github.io/Population_Genetics_in_R/workshop.html) will focus on the kinds of analyses typically conducted by plant pathologists. It will cover analyses of data from tools such as whole-genome variant calling and genotyping-by-sequencing or RADseq once variant call data are in hand. This workshop will not cover genome assembly, read mapping to reference genomes or the calling of variants. Participants will gain hands-on experience with analysis of VCF data in R using datasets provided by instructors. We will use R packages such as [`adegenet`](https://cran.r-project.org/web/packages/adegenet/index.html), [`poppr`](https://cran.r-project.org/web/packages/poppr/index.html), and [`vcfR`](https://cran.r-project.org/web/packages/vcfR/index.html). Basic familiarity with R is required."
abstract_short = "<img width=35% style = 'margin: 2px' src = '/img/learning/popgen2.png' align=right> This [Workshop](http://grunwaldlab.github.io/Population_Genetics_in_R/workshop.html) cover analyses of plan pathological data from tools such as whole-genome variant calling and genotyping-by-sequencing or RADseq once variant call data are in hand. This workshop will not cover genome assembly, read mapping to reference genomes or the calling of variants. "
event = "ICPP2018 July 28, 2018"
event_url = "https://apsnet.confex.com/apsnet/ICPP2018/meetingapp.cgi/Session/2042"
location = "Boston, United States"

# Is this a selected talk? (true/false)
selected = false

# Projects (optional).
#   Associate this talk with one or more of your projects.
#   Simply enter the filename (excluding '.md') of your project file in `content/project/`.
#projects = ["deep-learning"]

# Links (optional).
url_pdf = "http://grunwaldlab.github.io/Population_Genetics_in_R/index.html"
url_slides = ""
url_video = ""
url_code = "https://github.com/grunwaldlab/Population_Genetics_in_R/"

# Does the content use math formatting?
math = true

# Does the content use source code highlighting?
highlight = true

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]

+++
