---
title: "Evolution is not static: understanding factors that govern pathogen-host diversity"
description: |
  A post-talk interview with Dr. Remco Stam, a specialist on host-pathogen genetic diversity and (co)evolution. Remco is a forward-thinking researcher, with a humble yet passionate approach to understanding (co)evolution between host plants and their pathogens.
author:
  - name: Lisa Rothmann
    url: https://twitter.com/LandbouLisa
date: 04-16-2020
output:
  distill::distill_article:
    self_contained: false
preview: interview-remco.png
categories:
  - Interviews
---

Dr. [Remco Stam](http://www.remcostam.com/) ([@EvoMPMI](https://twitter.com/EvoMPMI)), a principal investigator (PI) at Technical University of Munich (Germany), delivered a talk on, _The diversity and molecular evolution of plant defence against pathogens in nature_ on the 6th of March at the [#OPPVirtualSeminars](https://openplantpathology.org/virtual_seminars/2020-04-06-remco-stam/).

### How did you find your experience presenting for the OPP Virtual Seminar? Have you done something similar in the past?  

I found presenting very exciting, but also a little strange because you are not seeing an audience like you normally would. I love public speaking and I love what I do, so I love telling people about it. I enjoy the interaction at conferences and seeing people's expressions and their body language, it tells you a lot about your talk and the delivery thereof. Virtually, you can not gauge the 'mood of the room' and you can not tell if people are distracted or on phones or making coffee... But, a positive is that you can judge from the questions asked and the feedback, that people did pay attention, and even more enjoyed the talk. I have given a virtual seminar before, with a department in Copenhagen, but there was a camera in the room with the audience, so there was some type of gauge for interactions and interest. I most definitely would give another talk in this format though.

### Any advice to future virtual seminar presenters?  

I would say my biggest piece of advice is - be yourself, like you always are. Keep true to yourself and present results you have confidence in, results you can validate with robust analyses. Some practical advice is to check your microphone and that you do not move away from your computer if you only have access to your computer's internal microphone. You can also find a way to grab people's attention, like a trigger, to bring them back to focusing on your presentation.


### How do you decide which population genetic statistics (evolutionary parameters) to select at the beginning of your study?

It is key that you can include all the parameters, as this provides the best opportunity to understand the whole system, with a slightly different lens for each parameter and in different combinations thereof too. Once you have collected your samples, you are bound to your data. Therefore it is key to think about the questions you want to be answered properly before sampling. You can use composites of the parameters measured and make inferences from calculating statistics from one another. But you always require robust data. [In our case](https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.16017), we realised that data on rare SNPs was not very reliable and some of the derived statistics would be over- or under-estimated. Then you need to make choices and find workarounds. As researchers we are also all limited by our budgets.

You have to make the best with what you have. You need to be clever in how you use the funds you are provided with to answer the most important questions. Ultimately, you need to select parameters which are going to enable you to do this. I read a lot of literature and my curiosity piqued. I was interested in attempting to "fill some of the gaps" of the literature related to population dynamics and how this relates to gene families.

### If I understand the work correctly, a higher non-synonymous diversity is observed in some NLRs across all the groups, what implication does this hold?

This is the whole point of figuring out the NLRs. On the one hand, we wanted to understand the evolutionary pressures and diversification of the genes. This is very important in some aspects of resistance in host plants because these are thought to adapt to new pathogens. Whereas, on the other hand, some genes are conserved, where no polymorphisms are observed, and these can also be vital to resistance. This paradox is what we need to understand the mechanisms underlying the purified or diversified selection. It is important that we know how plants in nature are "selecting" genes to assure pathogen recognition. Hopefully, the long term application is to see how these findings can be re-framed for resistance breeding or identifying candidate genes.

### Where many other researchers avoid outliers, it seems that outliers are important components of your research, I am making this assumption from additionally reading your preprint. Can you elaborate on this, perhaps with reference to the Fst indexes?

Evolution is not static, there is a lot of noise. A mutation might not mean anything, we may only see the significance thereof in later generations or by comparing a pattern we observe. This is why the outliers are so important to us, especially when we want to infer results from our Fst values. Usually, genes which are the most conserved or diverse are found outside of the normal distribution. This is an indication that the mutation was not a random chance but one of a process of selection. Of course, we filter out errors of the data set prior to assuming the outliers are significant so that we can inspect them further. One of the terms you can look into if these concepts are new to you is 'genetic drift'. We are interested in detecting if the loss of a gene was purely by chance or not.

### What is your approach to data analysis and analytical workflows?

As the PI, I do not prescribe a specific scripting language for our analyses. Everyone is free to do what they are comfortable with, as long as it works effectively. If you prefer Python over Perl, that is fine, I prefer Python and R, but it depends on your preference. We are standing on the shoulders of giants when it comes to selecting software. Gene sequencing and genome research has really advanced and taken a foothold over the last 20-years. We are an open community, where a lot of discussion and engagement is made to learn what are the best practices. Of course, there are risks, you need to keep level headed and make sure you share protocols and results you are confident in, that are validated. It is great to share and get your work out there. It is not only for your benefit and that people know you are reliable, but it is a platform where you can save people time, instead of struggling to put protocols in place that you have tried and tested.

### You mentioned fieldwork in Peru and Chile, why is this an important component of your research? How do you manage your field trips, and keep your collaborations vibrant and fruitfully functioning?

I am personally very interested in this, I think it is vital to go and collect samples from nature. I am really passionate about the evolution of resistance genes and the pathogen's response. We need people with local know-how they not only provide useful facilities but are also fantastic resources to navigate the terrain and internal protocols to move isolates between our countries. We adhere to the Nagoya Protocol, purifying the isolates in the country of origin and then only may we bring the cultures back to Europe. Our field trials are usually a period of two weeks, although they are an adventure, it can be exhausting. We wake up before sunrise, with the sun on the horizon, we drive long hours to make sure we get the distances we need to cover for the spatial aspects of the study. It is usually only three of us on a field trip, driving in a 4 x 4 across the landscapes. This is also associated with logistics and finances, if we are more people in the car we will require additional transport and these costs. The next phase, that I would love to do in the future, is looking at the severity and incidence of these pathogens, [which] would require a larger team - this should be interesting logistics to negotiate and implement.

### You mentioned, the potential need for plant physiologists to contribute further, how do you see this in the future?

Yes, in our studies we saw a lot of quantitative resistance. Where some plants exhibit a high level of infection but others do not display such strong responses to the pathogen. This means that there may be physiological processes involved, not just monogenic R-genes. We need to open up our research and reach out to people experienced outside our direct discipline. Sometimes in the field of Molecular Plant Pathology, as a community, we tend to overlook Plant Physiology. I think this is due to the influx of molecular work which can be done now, with such ease. All our fields have narrowed into niches, but I think our methods are converging and our data sets are becoming larger, providing an ideal situation to collaborate and interact more freely. This is what makes Plant Pathology so exciting for me, the interactions of disciplines and opportunities to learn.

### OPP: If you have to summarise your take away message in one sentence, how would you go about doing this?

I want to understand factors that govern coevolution between plants and pathogens, drawn from natural systems and how this extends into epidemics in our agronomic fields. This research also extends into fungicide resistance, and the artificial pressure or anthropogenic pressure placed onto pathogens to evolve. This is also a project we are working on Alternaria of potato, this pathogen has become a huge issue in Germany. Our long term goal aims to relieve the pressure of overusing or the reliance on fungicides, I think this is the goal of many Plant Pathologists to ensure an integrated pest management program.

### Thank you so much Remco for making the time to speak to me and to share your thoughts on coevolution and the underlying mechanisms. You were so patient and explained so clearly something which is outside my usual research 'comfort zone'. I am excited to see where your research will take you.

You can go and watch [Remco's #OPPvirtualseminar](https://www.youtube.com/watch?v=xaUz_68jwHk&feature=youtu.be) on our [official YouTube channel](https://www.youtube.com/channel/UCo-1ijIA_nECqzwzeW2X9RA?view_as=subscriber). You are also welcome to interact with us in our Slack Community on our #virtual-seminar channel. By clicking on this link you can [request to join](https://communityinviter.com/apps/openplantpathology/open-plant-pathology) Open Plant Pathology on Slack.






