---
title: "Workshop: Population Genomics in R"
description: |
  This workshop cover analyses of plan pathological data from tools such as whole-genome variant calling and genotyping-by-sequencing or RADseq once variant call data are in hand. This workshop will not cover genome assembly, read mapping to reference genomes or the calling of variants.
author:
  - name: Emerson Del Ponte
    url: https://twitter.com/edelponte
date: 08-06-2018
output:
  distill::distill_article:
    self_contained: false
preview: workshop-nik.png
categories:
  - Workshops
---

[Website](http://grunwaldlab.github.io/Population_Genetics_in_R/index.html) \| [Codes](https://github.com/grunwaldlab/Population_Genetics_in_R)

Taught at ICPP2018\
July 28, 2018 (1 to 5PM) Boston USA

Analysis of population genetic data remains challenging, particularly in the genomics era. High throughput sequencing has resulted in the opportunity to sequence variants (SNPs, indels, etc.) at the genome scale. However, application of these technologies for the characterization of pathogens provides unique challenges. This [workshop](http://grunwaldlab.github.io/Population_Genetics_in_R/workshop.html) will focus on the kinds of analyses typically conducted by plant pathologists. It will cover analyses of data from tools such as whole-genome variant calling and genotyping-by-sequencing or RADseq once variant call data are in hand. This workshop will not cover genome assembly, read mapping to reference genomes or the calling of variants. Participants will gain hands-on experience with analysis of VCF data in R using data sets provided by instructors. We will use R packages such as [`adegenet`](https://cran.r-project.org/web/packages/adegenet/index.html), [`poppr`](https://cran.r-project.org/web/packages/poppr/index.html), and [`vcfR`](https://cran.r-project.org/web/packages/vcfR/index.html). Basic familiarity with R is required.

![](workshop-nik.png)
